webapp/src/websocket/index.js

```diff
@@ -34,12 +35,26 @@ export function handleDisconnect(store) {
                 connected: false,
                 github_username: '',
                 github_client_id: '',
-                settings: {},
+                user_settings: {},
+                configuration: {},
             },
         });
     };
 }
 
+export function handleConfigurationUpdate(store) {
+    return (msg) => {
+        if (!msg.data) {
+            return;
+        }
+
+        store.dispatch({
+            type: ActionTypes.RECEIVED_CONFIGURATION,
+            data: {...msg.data},
```

Comment:
Seems weird to do this in this case 0/5

```suggestion
            data: msg.data,
```